locale/de/LC_MESSAGES/django.po

```diff
@@ -281,12 +286,13 @@ msgstr "Zugang zum Projekt"
 
 #: adhocracy4/projects/models.py:65
 msgid ""
-"Please indicate who should be able to participate in your project. Teasers "
-"for your project including title and short description will always be "
-"visible to everyone"
+"Please indicate whether this project should be public or restricted to "
+"invited users. Teasers for your project including title and short "
+"description will always be visible to everyone"
 msgstr ""
-"Bitte geben Sie an, ob das Projekt öffentlich zugänglich sein soll oder ob "
-"nur eingeladene Nutzer*innen teilnehmen können."
+"Bitte geben Sie an, ob dieses Projekt öffentlich oder beschränkt auf "
+"eingeladene Teilnehmerinnen und Teinehmer sein soll. Kacheln für Ihr Projekt "
+"mit Titel und Kurzbeschreibung werden immer für alle sichtbar sein."
```

Comment:
Tiles of private projects are shown only to invited users.

Grammar: "öffentlich oder auf eingeladene ... berschränkt"